static/js/checks.js

```diff
@@ -4,7 +4,8 @@ $(function () {
     var HOUR = {name: "hour", nsecs: MINUTE.nsecs * 60};
     var DAY = {name: "day", nsecs: HOUR.nsecs * 24};
     var WEEK = {name: "week", nsecs: DAY.nsecs * 7};
-    var UNITS = [WEEK, DAY, HOUR, MINUTE];
+    var MONTH = {name: "month", nsecs: DAY.nsecs * 30};
```

Comment:
Wouldn't it better if 
`var MONTH = {name: "month", nsecs: WEEK.nsecs * 4}` to incorporate the varying number of days in the different months

Reply:
@WNjihia The Weekly unit is equal to 7 days (a value which remains constant), 7*4 = 28 days doesn't cover the number of varying days in various months. I used 30 days considering it's the standard value mostly used to represent a monthly period.

Reply:
That does make sense. Let's stick with what you have